README.rst

```diff
@@ -166,6 +166,10 @@ There are multiple supported backends, which can be selected at runtime in the e
 +----------------------------+---------------------------------------------------+-----------------------+
 | ``/gpu/cuda/magma/det``    | CUDA MAGMA kernels                                | Yes                   |
 +----------------------------+---------------------------------------------------+-----------------------+
+| ``/gpu/hip/magma``         | HIP MAGMA kernels                                 | No                    |
++----------------------------+---------------------------------------------------+-----------------------+
+| ``/gpu/hip/magma/det``     | HIP MAGMA kernels                                 | Yes                   |
++----------------------------+---------------------------------------------------+-----------------------+
 | HIP Native Backend                                                                                     |
 +----------------------------+---------------------------------------------------+-----------------------+
 | ``/gpu/hip/ref``           | Reference pure HIP kernels                        | Yes                   |
```

Comment:
Minor - since MAGMA now has hip and cuda backends, perhaps we wish to list MAGMA on the README after both the cuda and hip native backends?

Reply:
I agree, that makes sense.  I swapped the order in 655f042f0c35c6c44ac9ff1e23c365ef0001ac32